File: app/src/main/java/com/cookplan/recipe/edit/add_info/EditRecipeInfoPresenter.java
package com.cookplan.recipe.edit.add_info;

import android.net.Uri;

import com.cookplan.models.Recipe;

/**
 * Created by DariaEfimova on 16.03.17.
 */

public interface EditRecipeInfoPresenter {

    public Uri getOutputImagePath();

    public void doOCR(String language);

    public void saveRecipe(Recipe recipe, String newName, String newDesc);
}

File: app/src/main/java/com/cookplan/recipe/edit/add_info/EditRecipeInfoPresenter.kt
package com.cookplan.recipe.edit.add_info

import android.net.Uri

import com.cookplan.models.Recipe

/**
 * Created by DariaEfimova on 16.03.17.
 */

interface EditRecipeInfoPresenter {

    fun getOutputImagePath(): Uri?

    fun doOCR(language: String)

    fun saveRecipe(recipe: Recipe?, newName: String, newDesc: String)
}
